Makefile

```diff
@@ -122,7 +126,8 @@ ChangeLog: VERSION
 tag:
 	@echo "** Tagging the release **  1) pull existing;  2) tag  3) push it"
 	git pull --tags
-	git tag -a -m'release tagging' v$(ESSVERSION)
+	@echo "Creating tag and signing using gpg"
+	git tag -s -m'release tagging' v$(ESSVERSION)
```

Comment:
Don't we have to push the tags explicitly or the `git push` from below does the job?

Reply:
I believe the git push does that.